scripts/merge-conflicts.sh

```diff
@@ -0,0 +1,30 @@
+originalUrl="https://github.com/ethereum/solidity.git"
```

Comment:
Can you add some `#!/bin/sh` headers and such?
Also `set -sane options`?

Reply:
actually `git pull` returns non-zero code when finding any conflict.

Reply:
We do something like `git pull ... || true` in that case. Overall we prefer to have at least `set -e` in our scripts whenever possible because having them fail by default is much more intuitive and you can always override it with `|| true`.